logger.go

```diff
@@ -45,6 +45,8 @@ func (l *eventLogger) Write(p []byte) (int, error) {
 func Logf(ctx context.Context, f string, args ...interface{}) {
 	logger := LoggerFromContext(ctx)
 	if logger == nil {
+		fmt.Printf("🟡 Warning: There was no logger found in context, so falling back to `fmt.Printf`")
```

Comment:
This does not print a newline character, so this and the next printf print to the same line. Is that intentional?
If not, fmt.Println might be more appropriate for the first call, since the first call also doens't use any formatting.

Reply:
Thanks for catching, updated!